src/averageAges.js

```diff
@@ -14,12 +14,14 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = false) {
+  const males = people.filter((male) => {
+    return century === false
+      ? male.sex === 'm'
+      : (male.sex === 'm' && Math.ceil(male.died / 100) === century);
+  });
```

Comment:
depending on the case, we return the appropriate callback

```suggestion
  const males = people.filter(
    century
      ? male => male.sex === 'm'
      : male => male.sex === 'm'
        && Math.ceil(male.died / 100) === century;
  );
```